### DISCUSSION TITLE
Create pattern for new releases on the Admin

### DISCUSSION THREAD

**Discussion by beatrizmilhomem** [vtex/shoreline#955] [Design RFCs]
### Motivation
Teams create different solutions for new releases on the Admin to onboard users. We should create a pattern for teams to use.

### Examples

- Modal from Orders redesign
- Delivery options page

![Image](https://github.com/vtex/shoreline/assets/84085258/b474de36-a2db-489b-a9ad-c95baa643a44)

### Explorations
[Figma](https://www.figma.com/file/3mwXMX5fxYIWbL8bT5oq7A/Smart-Help?type=design&node-id=63-20843&mode=design&t=3GVVjqQp3vUlLNJh-4)

**Comment by davicostalf** [comment]
I think these type of suggestion should be a discussion under Design RFCs and not an issue yet

**Reply by beatrizmilhomem** [reply]
Done ✅
